src/libraries/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -938,7 +938,7 @@ public void ProcessStartTime_Deterministic_Across_Instances()
             {
                 using (var p = Process.GetProcessById(_process.Id))
                 {
-                    Assert.Equal(_process.StartTime, p.StartTime);
+                    Assert.Equal(_process.StartTime, p.StartTime, precision: TimeSpan.FromMilliseconds(10));
```

Comment:
This is not needed anymore (since https://github.com/dotnet/runtime/pull/67589#discussion_r867885466 was addressed):
```suggestion
                    Assert.Equal(_process.StartTime, p.StartTime);
```

Reply:
> I will apply this once current test run is completed on Android; this test doesn't run on Android so it doesn't need a rerun.

@am11 what is the status of this?

Reply:
Tests failed on Android. I had to add an explicit cast. Could you please re-run `runtime-extra-platforms` pipeline?